src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
```

Comment:
you could use mixin here, but I think it's not really important =)

Reply:
Really, i'm forget about it, will change this soon. Thx